proto/poktroll/supplier/event.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package poktroll.supplier;
+
+option go_package = "github.com/pokt-network/poktroll/x/supplier/types";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+import "poktroll/shared/supplier.proto";
+
+// EventSupplierStaked is emitted with the commitment of the supplier is stake message.
+message EventSupplierStaked {
+  poktroll.shared.Supplier supplier = 1;
+}
+
+// EventSupplierUnstaked is emitted with the commitment of the supplier is unstake message.
+message EventSupplierUnbondingBegin {
+  poktroll.shared.Supplier supplier = 1;
```

Comment:
Would be interesting to include the unbonding height here (If it's not too difficult to calculate it).